shared/common-adapters/usernames/index.tsx

```diff
@@ -88,7 +88,7 @@ const UsernameText = (props: Props) => {
         const _onUsernameClicked = props.onUsernameClicked
         const isNegative = backgroundModeIsNegative(props.backgroundMode || null)
         const renderText = (onLongPress?: () => void) => (
-          <Text type="Body" key={u.username}>
+          <Text type="BodyBold" key={u.username}>
```

Comment:
I think you want this on the inner `Text` component. Otherwise, `,` and `and` will be bold too.

Reply:
👍🏻fixed